CSharp/BatchExplorer/readme.txt

```diff
@@ -0,0 +1,11 @@
+﻿Once the app is running, you need to configure it with data from your Azure subscription.
+
+1. On the Accounts menu click "Add Default Account Manager". The "Add Account" dialog opens.
+2. In the Account Alias box, type in any name you like. Use the name to create distinct batch jobs.
+3. For the Batch Service Url, go to your subscription and copy the URL. Paste it into the box.
+4. For the Account box, go to your subscription and copy the name of the Batch. Paste it into the box.
```

Comment:
For the Account box, go to your subscription and copy the name of the Batch Account. Paste it into the box.